SystemServices/CHANGELOG.md

```diff
@@ -28,6 +28,10 @@ All notable changes to this RDK Service will be documented in this file.
 ### Security
 - resolved security vulnerabilities
 
+## [1.7.4] - 2023-01-04
```

Comment:
Revision cannot go backwards. Pls update patch version and move it to 2.0.3. You also need to update the version macros for the system plugin - https://github.com/rdkcentral/rdkservices/blob/main/SystemServices/SystemServices.cpp#L70